File: angular17_web/src/app/models/product-color/update.product.color.ts
export class UpdateProductColor {
  name?: string;
  hexCode?: string;
  stock?: number;

  constructor(data?: any) {
    if (!data) return;

    this.name = data.name;
    this.hexCode = data.hexCode;
    this.stock = data.stock;
  }
}

File: Electronics_Shop_17.Model/Requests/UpdateProductColor.cs
﻿using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Electronics_Shop_17.Model.Requests
{
    public class UpdateProductColor
    {
        public string? Name { get; set; }
        public string? HexCode { get; set; }
        public int? Stock { get; set; }
    }
}
